python/GafferUI/WidgetAlgo.py

```diff
@@ -47,11 +47,11 @@
 def joinEdges( listContainer ) :
 
 	if listContainer.orientation() == listContainer.Orientation.Horizontal :
-		lowProperty = "gafferFlatLeft"
-		highProperty = "gafferFlatRight"
+		lowProperty = "gafferAdjoinedLeft"
```

Comment:
I don't know where in the chain this has got broken, but the joining is no longer working in the SceneInspector. Make two nodes, select them both, and then look at the "Node" section at the top of the SceneInspector. For me, it's no longer joined.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/3229/commits/7adb3990b829a0825a197f5049f9a0ff0c04f20c

Reply:
Thanks. Somewhere along the line a one pixel gap has also appeared between the two widgets in the Node section - is that intentional? The frames for the Globals->Sets view in the SceneInspector have also gained a border that seems out of keeping with the rest of the scene inspector.

![image](https://user-images.githubusercontent.com/1133871/60650916-ff4d6780-9e3c-11e9-9137-7ca98e5c2a79.png)

Nearly there!

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/3229/commits/db14932b47aeb8fa371745df097578e5e3856f28 and https://github.com/GafferHQ/gaffer/pull/3229/commits/125f3ae21d471a40d5ed487f8ebc41062714beae